tests/st/runtime/control_flow/test_dyn_orch_shape.py

```
@@ -291,7 +291,7 @@ def compute_expected(self, tensors, params=None):
 
 
 class DynOrchValidShapeAddTestCase(PTOTestCase):
-    """Test add with dynamic M×N orchestration and valid_shapes from a scalar tensor.
+    """Test add with dynamic M×N orchestration and valid_shape from a scalar tensor.
```

Ruff (0.16.0)

[warning] Docstring contains ambiguous `×` (MULTIPLICATION SIGN). Did you mean `x` (LATIN SMALL LETTER X)?

(RUF002)